handlers/analytics/log_processor.go

```diff
@@ -135,9 +135,22 @@ func (lp *LogProcessor) Start(ch chan LogData) {
 }
 
 func (p *LogProcessor) processLog(d LogData) {
+	updateMetrics(d)
 	p.logs = append(p.logs, d)
 }
 
+func updateMetrics(d LogData) {
+	if d.EventType != "heartbeat" {
+		return
+	}
+	metrics.Metrics.AnalyticsMetrics.AnalyticsLogsPlaytimeMs.
+		WithLabelValues(d.PlaybackID, d.UserID, d.PlaybackContinentName).
+		Observe(float64(*d.EventData.TimePlayingMS))
+	metrics.Metrics.AnalyticsMetrics.AnalyticsLogsBufferTimeMs.
+		WithLabelValues(d.PlaybackID, d.UserID, d.PlaybackContinentName).
+		Observe(float64(*d.EventData.TimeStalledMS + *d.EventData.TimeWaitingMS))
```

Comment:
@leszko 
https://eu-metrics-monitoring.livepeer.live/grafana/goto/PIa9O1sIg?orgId=1

Reply:
New analytics is not yet anabled in prod. I think we plan to enable it this or next week. 

In staging, I see it works: https://eu-metrics-monitoring.livepeer.monster/grafana/goto/6iDmK1sSg?orgId=1

Reply:
https://eu-metrics-monitoring.livepeer.live/grafana/d/cdsxag32xzojkf/rebuffer-rate?orgId=1 finally in prod... but 7-8% rebuffer rate globally seems really high

Reply:
I think we should reconsider to use [`NewMetricWithTimestamp()`](https://pkg.go.dev/github.com/prometheus/client_golang/prometheus#NewMetricWithTimestamp) to publish metrics with the timestamps from the events and not `now()`.

Reply:
@leszko wdyt?

Reply:
Good point. Though, if there is no heartbeat retries from the player (and I believe there should not be), then it should not differ by more than 5s. So, maybe it's ok to keep it as it is.

Reply:
> Good point. Though, if there is no heartbeat retries from the player (and I believe there should not be), then it should not differ by more than 5s. So, maybe it's ok to keep it as it is.

In case of 5XX... and FE retrying to to push the events... the difference might grow...

In general I was looking for the source of differences between these prometheus metrics and Clickhouse rebufer... but maybe 5s isn't the main source of them